subprojects/build-scan-performance/build-scan-performance.gradle.kts

```diff
@@ -37,9 +30,9 @@ dependencies {
 
     performanceTestImplementation(project(":internalIntegTesting"))
 }
-
-gradlebuildJava {
-    moduleType = ModuleType.INTERNAL
+// so that all Gradle features are available
+configurations.performanceTestRuntimeOnly.get().withDependencies {
+    addAll(rootProject.configurations.testRuntime.get().allDependencies)
```

Comment:
I made this change in some places to get rid of the "extra" properties set in the root build script.

In the next step I want to clean up the distribution building and improve this part further so that we do not need to reach into the root project anymore.